src/test/resources/application.conf

```diff
@@ -245,19 +245,37 @@ cnp_perftest {
   oAuth2clientSecret = "${IDAM_OAUTH2_CLIENT_SECRET}"
 
   cases = ["1559642192860544"]
-  getCaseUrl = "internal/cases/:case_reference"
+  ccdCaseMgmtUrl = "https://ccd-case-management-web-perftest.service.core-compute-perftest.internal"
+  getCaseUrl = "aggregated/caseworkers/a2fe2e87-94f7-44a9-a272-5c413a4775c0/jurisdictions/AUTOTEST1/case-types/AAT/cases/1531212825013102"
+  getCaseUrlV2 = "internal/cases/1559642192860544"
   createCaseUrl = "caseworkers/a2fe2e87-94f7-44a9-a272-5c413a4775c0/jurisdictions/:jurisdictions_reference/case-types/:casetype_reference/cases"
   validateCaseUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases"
-  getUserProfileUrl = "caseworkers/176475/profile"
+  getUserProfileUrl = "caseworkers/a2fe2e87-94f7-44a9-a272-5c413a4775c0/profile"
+  getUserProfileUrlV2 = "caseworkers/539560/profile"
   searchCases = "aggregated/caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases?state=CaseCreated"
   getPaginationMetadataUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases/pagination_metadata?state=CaseCreated"
   saveEventUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/CASETYPE2/cases/1531207599597587/events"
   searchInputDetails = "aggregated/caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/inputs"
   workbasketInputDetails = "/aggregated/caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/work-basket-inputs"
   getPrintableDocumentsForEvent = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/CASETYPE2/cases/1531207599597587/documents"
   validateCaseDetails = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/validate"
-  docStoreBashURL = "https://dm-store-perftest.service.core-compute-perftest.internal"
   ESSearch = "searchCases"
+  docStoreBashURL123 = "https://dm-store-perftest.service.core-compute-perftest.internal"
+  docStoreBashURL = "https://gateway-ccd.perftest.platform.hmcts.net"
+  prodSSCSWorkBasket = "aggregated/caseworkers/560966/jurisdictions/SSCS/case-types/Benefit/cases?view=WORKBASKET&page=1&case.caseReference=test1234567"
+  prodSSCSPagination = "caseworkers/560966/jurisdictions/SSCS/case-types/Benefit/cases/pagination_metadata?case.caseReference=test123456"
```

Comment:
`560966` used in multiple places. Duplication problem. Perhaps extract in a variable and reuse it?